roles/yoda_portal/defaults/main.yml

```diff
@@ -32,6 +29,14 @@ openssl_certs_dir: '/etc/pki/tls/certs'
 openssl_key_signed: localhost.key
 openssl_crt_signed_and_chain: localhost_and_chain.crt
 
+# Flask configuration
+flask_secret_key: PleaseChangeMe123
+flask_portal_title: Yoda Portal
```

Comment:
We also have the portal_title_text configuration? Is this redundant?

Reply:
Indeed redundant. Replaced with original portal_title_text in other files and made the default 'Yoda Portal', for the development/allinone environment the default is 'Yoda - Dev'.
The other old portal variables need to be checked as well if they can be deleted.